### ISSUE TITLE
How to use Helium_M1_v0.1.2.iso?

### ISSUE THREAD

**Issue by probonopd** [mszoek/helium#43]
Tried running `Helium_M1_v0.1.2.iso` on a machine with a Nvidia GPU. Drops me to a non-GUI login prompt where I can log in as root, but there is no `startx` command nor `Xorg` that I could find...

**Comment by mszoek** [comment]
Xorg and startx should be in /usr/bin. If they aren't then something is wrong with that image.

**Comment by probonopd** [comment]
Can't find them there :-/

**Comment by mszoek** [comment]
Grrrrr..ok. Sorry about that! You should be able to install anything missing with `pkg install` for now. I will look into what happened and fix the image!

**Comment by probonopd** [comment]
I'd be especially interested in [that "special" version of libjpeg](https://github.com/mszoek/helium/issues/41)... :)

**Comment by mszoek** [comment]
Weird! I downloaded the Helium_M1_v0.1.2 iso from GitHub and compared it to my build output. They have the same SHA sum and the helium.txz on the ISO does contain both Xorg and startx.  Installed it to a fresh VM and ran `startx` - no issues. I get the disgusting twm display I expected to see :)

**Did you by any chance unselect the 'helium' package from the installer?**

<img width="1287" alt="image" src="https://user-images.githubusercontent.com/60144291/116005647-eabffd80-a5d5-11eb-8104-2713f353d388.png">

`/usr/lib/libjpeg.so.*` is also there, as it should be. (Incidentally, it is not particularly special. It's built from ports: `graphics/jpeg-turbo`. It looks like the port was updated since I built it, so now it produces libjpeg.so.8.2.2)
